subgraph/bin/thegraph

```diff
@@ -25,30 +29,26 @@ const executeCommand = async (command) => {
   }
 }
 
-const incrementVersion = (version) => {
-  const [_, major, minor, patch] = version.match(
-    /v([0-9]*)\.([0-9]*)\.([0-9]*)/
-  )
-  return `v${major}.${minor}.${parseInt(patch) + 1}`
+const getVersion = () => {
+  const gitSha = require('child_process')
+    .execSync('git rev-parse --short HEAD')
+    .toString()
+    .trim()
+  return `v${UNLOCK_LATEST_VERSION}-${PUBLICLOCK_LATEST_VERSION}-${gitSha}`
```

Comment:
maybe commit hash alone is simpler ?

Reply:
Yeah, in the end the unlock and public lock versions are not that useful anyway because the subgraoh supports them all at all times.